uint32 size
float64 min_distance
geometry_msgs/Point initial
geometry_msgs/Point centroid
geometry_msgs/Point middle
